### DISCUSSION TITLE
Does this version automatically support OneDrive sync to Kobo devices?

### DISCUSSION THREAD

**Discussion by victor-marino** [marklar423/KoboCloud-rclone#4] [Q&A]
Hi there!

I arrived at this fork from the original repo, and although I think this is implied by the changes you made to the readme, I just wanted to confirm:

If I install and configure this version on my Kobo, will I be able to sync OneDrive folders straightaway?

Thanks!

**Comment by mealilo** [comment]
If I understand this repo correctly, anything that rclone can get, will work with this repo.


Here's a link from rclone's website on how to setup onedrive

https://rclone.org/onedrive/
